### PR TITLE
serve static middleware for node

### PR DESCRIPTION
**Pull request by that-ambuj**
As mentioned in issue #3, It is the serve static implementation for hono node server.
I have also added hono's repo as dev dependency and linked it to easily import types and utils from the main repo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,12 +33,12 @@
     "@types/jest": "^29.0.1",
     "@types/node": "^18.7.16",
     "@types/supertest": "^2.0.12",
-    "hono": "^2.1.4",
+    "hono": "https://github.com/honojs/hono",
```
Comment (yusukebe) [inline package.json:36]: Please use "hono" from the npm registry. Not github.com

**Review by yusukebe — CHANGES_REQUESTED** [review]

**Comment by yusukebe** [comment]
By importing hono, the tests are failing and I will investigate.

**Comment by that-ambuj** [comment]
Please close this PR, I have made correct changes in PR #5

File: src/serve-static.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { existsSync, readFileSync } from 'fs'
+import type { MiddlewareHandler } from 'hono/src/hono'
```
Comment (yusukebe) [inline src/serve-static.ts:2]: Fix path: `hono/src/hono` to `hono` etc.

**Review by yusukebe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**